pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -250,10 +255,6 @@ func (adc *attachDetachController) podAdd(obj interface{}) {
 	if pod == nil || !ok {
 		return
 	}
-	if pod.Spec.NodeName == "" {
```

Comment:
It's redundant with ProcessPodVolumes, which does the same check (with a log message).